### ISSUE TITLE
Add OMPD model as a first class model in GC

### ISSUE THREAD

**Issue by liversedge** [GoldenCheetah/GoldenCheetah#3366]
Add support for the multi-domain model in GC, as a first class model (i.e. available in all places where the CP2, CP3 and extended models are).

See: https://veloclinic.com/development-and-field-validation-of-an-omni-domain-power-duration-model/

**Comment by liversedge** [comment]
It may be more appropriate to add this as an option to the CP3 model (W'eff and decay).
This will need to be thought thru as currently the models are not configurable (e.g. when discovering estimates at startup).

**Comment by amtriathlon** [comment]
What about to use OMPD as a replacement for Extended CP?

**Comment by liversedge** [comment]
Nah, just add as a new one. Might add Perennet-Thibault too haha!
